build.sbt

```diff
@@ -148,7 +148,7 @@ lazy val tester = (project in file("tester"))
     version := SpinalVersion.tester,
     baseDirectory in (Test) := file("./"),
 
-    libraryDependencies += "org.scalatest" % "scalatest_2.11" % "2.2.1",
+    libraryDependencies += "org.scalatest" % "scalatest_2.12" % "3.1.0",
```

Comment:
Can't update to 2.12.x, see the review comment about the SpinalVersion.scala file :)
More generaly, i try to avoid updating depedancies if that's not required, as some people use SpinalHDL ofline +  there is not so much value updating something which work

Can you revert those changes ?